### DISCUSSION TITLE
Not working dark mode with prefix

### DISCUSSION THREAD

**Discussion by dmitryburov** [tailwindlabs/tailwindcss#5476] [Help] [answered]
Hello everyone!
I'm using NuxtJS(v2.15.8) with TailwindCSS (v2.2.14) and set `prefix` param.
Light mode - work, but dark mode is fail.

See my `tailwindcss.config.js`:

![image](https://user-images.githubusercontent.com/11191484/132947410-c28af874-4017-4b4b-92b2-fb3d0c3d8b07.png)

```html
<div class="fx-bg-white dark:fx-bg-gray-900">
   ...
</div>
```

If delete prefix... it's work! =)
What wrong?(((

**Comment by dance2die** [comment, marked as answer]
When you set the `darkMode: 'class'`, `dark` class should be applied to the containing element.

e.g. Applied to the direct parent: this works
**Demo:** https://play.tailwindcss.com/fjMwOCw0sN
```html
<div class="dark">
  <div class="min-h-screen dark:bg-gray-600 bg-gray-100 py-6 flex flex-col justify-center sm:py-12">
...
```

This won't work
**Demo:** https://play.tailwindcss.com/8YTQN2pHLL
```html
<!--                     👇  -->
<div class="min-h-screen dark dark:bg-gray-600 bg-gray-100 py-6 flex flex-col justify-center sm:py-12">
```

I don't see a `dark` class applied in your code so could you check if it's applied in the containing element?

**Reply by dmitryburov** [reply]
Oh no...
Where this info in docs? I didn't see it(

You example is works!
Tnx @dance2die !

**Reply by dance2die** [reply]
yw there.  

> Where this info in docs? I didn't see it(

Check out https://tailwindcss.com/docs/dark-mode#toggling-dark-mode-manually
